rules/type-declaration/tests/Rector/FunctionLike/ParamTypeDeclarationRector/Fixture/vat_rate.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\ParamTypeDeclarationRector\Fixture;
+
+interface SimpleValueObjectInterface
+{
+    /**
+     * @param float|int|string $value
+     */
+    public function __construct($value);
+}
+
+interface VatRateInterface extends SimpleValueObjectInterface
+{
+    /**
+     * @param string $countryCode
+     */
+    public function __construct($countryCode);
+}
+
+final class VatRate implements VatRateInterface
```

Comment:
class name can be changed with `Skip` prefix, eg:

```suggestion
final class SkipDifferentWithInterface implements VatRateInterface
```

and fixture file name can be changed to `skip_different_with_interface.php.inc`

Reply:
Done!